example_data/rnaseq_star/rnatest_star_fusion.yaml

```diff
@@ -0,0 +1,37 @@
+instrument_data_bams:
+    - class: File
+      path: /gscmnt/gc2733/walter/sridnona/softwares/rnaseq_mgbio/test.bam
+    - class: File
+      path: /gscmnt/gc2733/walter/sridnona/softwares/rnaseq_mgbio/test2.bam
+trimming_adapters:
+    class: File
+    path: /gscmnt/gc2560/core/illumina_adapters/illumina_multiplex.fa
+trimming_adapter_trim_end: 'RIGHT'
+trimming_adapter_min_overlap: 7
+trimming_max_uncalled: 300
+trimming_min_readlength: 25
+sample_name: H_NJ-HCC1395-HCC1395_RNA
+kallisto_index:
+    class: File
+    path: /gscmnt/gc2560/core/GRC-human-build38_human_90_38/rna_seq_annotation/Homo_sapiens.GRCh38.cdna.all.fa.kallisto.idx
+gene_transcript_lookup_table:
+    class: File
+    path: /gscmnt/gc2560/core/GRC-human-build38_human_90_38/rna_seq_annotation/ensembl90.transcriptToGene.tsv
+strand: first
+
+outSAMattrRGline:
+    - "ID:2895626107 PU:H3MYFBBXX5.4.GCCAAT SM:H_NJ-HCC1395-HCC1395_RNA LB:Pooled_RNA_2891007020-cD1-lib1 PL:Illumina CN:WUGSC"
+    - "ID:2895626112 PU:H3MYFBBXX5.5.GCCAAT SM:H_NJ-HCC1395-HCC1395_RNA LB:Pooled_RNA_2891007020-cD1-lib1 PL:Illumina CN:WUGSC"
+
+gtf_file:
+    class: File
+    path: /gscmnt/gc2733/walter/sridnona/refs/stock_refs/star/human_38_90/Homo_sapiens.GRCh38.90.gtf
```

Comment:
We've already talked about the need to get these indices and anno files somewhere central. My suggestion would be to add the needed instructions to this wiki page:
https://github.com/genome/analysis-workflows/wiki/Creating-RNAseq-annotation-files-from-Ensembl

Then we'll backfill existing annotation builds, and make sure that the STAR-relevant files are created for future builds.

Reply:
cloning wiki.git and pushing it is a bit trivial, can some one who has access to it add few lines on how to build star index
```
#make STAR index
#NOTE - STAR version 2.7.0f
LSF_DOCKER_PRESERVE_ENVIRONMENT=false bsub -J star_index -n 10 -q research-hpc -oo star_index.o -eo star.index.e -R"select[mem>64000] rusage[mem=64000]" -M 64000000 -a "docker(mgibio/star)" /bin/bash -c "STAR --runMode genomeGenerate --runThreadN 10 --genomeDir star/human_38_95 --genomeFastaFiles Homo_sapiens.GRCh38.dna.primary_assembly.fa --sjdbGTFfile Homo_sapiens.GRCh38.95.gtf --sjdbOverhang 100"
```

Reply:
Done!